### PR TITLE
Task 4 course card

### PR DESCRIPTION
**Pull request by OlegPanasyuk**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,21 +17,21 @@
...
+    "eslint-config-prettier": "^6.4.0",
+    "eslint-config-react-app": "^5.0.2",
     "eslint-plugin-flowtype": "3.13.0",
     "eslint-plugin-import": "2.18.2",
-    "eslint-plugin-jsx-a11y": "6.2.3",
-    "eslint-plugin-prettier": "3.1.0",
-    "eslint-plugin-react": "7.14.3",
+    "eslint-plugin-jsx-a11y": "^6.2.3",
+    "eslint-plugin-prettier": "^3.1.1",
+    "eslint-plugin-react": "^7.16.0",
```
Comment (slavasgod) [inline package.json]: Remove `^` from all packages

File: src/components/courseCard/index.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+import Rating from '../rating/index';
+
+import styles from './style.module.scss';
+
+import { links } from '../../utils/constants';
+import Img from '../img/index';
+
+export default function CourseCard({ item }) {
+  const { title, author, rate, price, img = '', id = 1, votes } = item;
+
```
Comment (slavasgod) [inline src/components/courseCard/index.js:22]: Import order + prop-types + export const(`export function` in your case)

File: src/components/img/index.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    });
+  };
+
+  onError = () => {
+    const image = this.refImg.current;
+    this.setState({
+      isLoading: false,
+    });
+    image.src = 'http://placeimg.com/640/480/tech';
+  };
```
Comment (slavasgod) [inline src/components/img/index.js:38]: Looks like hardcode. Move it to constants or download this picture to assets

**Review by slavasgod — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (OlegPanasyuk) [reply package.json]: Done

File: src/components/courseCard/index.js (hunk 2 above)
Comment (OlegPanasyuk) [reply src/components/courseCard/index.js:22]: Done

File: src/components/img/index.js (hunk 3 above)
Comment (OlegPanasyuk) [reply src/components/img/index.js:38]: done

**Review by slavasgod — APPROVED** [review]

File: src/components/img/index.js [hunk 4]
```diff
@@ -0,0 +1,50 @@
+import React, { Component } from 'react';
+import Skeleton from 'react-loading-skeleton';
+import { links } from '../../utils/constants';
```
Comment (melonholy) [inline src/components/img/index.js]: pls make spaces between the imports

Comment (OlegPanasyuk) [reply src/components/img/index.js]: Done

**Review by melonholy — APPROVED** [review]

**Pull request merged**